tests/unit/test_clidriver.py

```diff
@@ -261,6 +261,15 @@ def test_error_logger(self):
         expected = {'log_level': logging.ERROR, 'logger_name': 'awscli'}
         self.assertEqual(driver.session.stream_logger_args[1], expected)
 
+    def test_ctrl_c_is_handled(self):
+        driver = CLIDriver(session=self.session)
+        fake_client = mock.Mock()
+        fake_client.list_objects.side_effect = KeyboardInterrupt
+        fake_client.can_paginate.return_value = False
+        driver.session.create_client = mock.Mock(return_value=fake_client)
+        rc = driver.main('s3 list-objects --bucket foo'.split())
+        self.assertEqual(rc, 130)
+
```

Comment:
Could we could possibly use BaseAWSCommandParamsTest with a mocked session and the ``run_cmd()`` method to capture the stderr?